CollAction/Frontend/app/global/MobileNavDraw.tsx

```diff
@@ -36,10 +53,8 @@ export default class MobileNavDraw extends React.Component<IMobileNavDrawProps,
             <li><a href="/find">Find Project</a></li>
             <li><a href="/start">Start Project</a></li>
             <li><a href="/about">About</a></li>
-            {this.props.loginText == "Logout" ?
-                (<li><button type="button" onClick={() => document.getElementById("logOutBtn").click()}>Logout</button></li>) :
-                (<li><a href="/account/login">Login</a></li>)
-            }
+            <AccountManageNav isLoggedIn={this.props.isLoggedIn} />
```

Comment:
This works, but there's a much simpler solution, inside render, add this variable:

```
let accountHtml;

if (this.props.isLoggedIn) {
    accountHtml = <li><a href="/manage">Account</a></li><li><button type="button" onClick={() => document.getElementById("logOutBtn").click()}>Logout</button></li>;
} else {
    accountHtml = <li><a href="/account/login">Login</a></li>;
}
```

and then

```
return(
      <div id="draw-menu">
        <div id="draw-menu-close-button" onClick={() => self.close()}>
          <i className="fa fa-times" aria-hidden="true"></i>
        </div>
        <ul>
            {accountHtml}
        </ul>
      </div>
);
```

I'm fine with the component based solution, this is a lot simpler though.